client-angular/custom-webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const { resolveDefineConfig } = require('@genesislcap/build-kit');
+const { config } = require('dotenv');
+const { DefinePlugin } = require('webpack');
+const pkg = require('./package.json');
+const pkgConfig = Object.fromEntries(
+    Object.entries(pkg.config).map(([key, value]) => [key, JSON.stringify(value)])
+);
+config();
+
+module.exports = {
+  plugins: [
+    new DefinePlugin(resolveDefineConfig(['API_HOST', 'GENX_*', 'DEFAULT_USER', 'DEFAULT_PASSWORD'])),
+    new DefinePlugin(pkgConfig),
```

Comment:
We don't want API_HOST and default user/pwd in prod build, only dev one. From memory we load `GENX_*` for both dev and prod, the rest is only loaded if you ask for it (by passing `-e`).

Reply:
changed, let us know if it's ok now

Reply:
at the moment we are using foundation-login which [uses API_HOST for defaultApiHost](https://github.com/genesislcap/foundation-ui/blob/6821e0bf24c55b54d8c251b70ef22859470ed006/packages/foundation/foundation-mf/foundation-login/src/main/config.ts#L34)

so I think we can leave it here (API_HOST) now and remove it later when we implement foundation-auth